src/components/Nav.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable implicit-arrow-linebreak */
+import { React, useState, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// import { NavLink } from 'react-router-dom';
```

Comment:
- [ ] Please, kindly remove this old piece of commented code, to improve the cleanliness of the repo 💪